Wrappers/Python/ccpi/framework.py

```diff
@@ -720,6 +720,10 @@ def sqrt(self, out=None, *args,  **kwargs):
     ## reductions
     def sum(self, out=None, *args, **kwargs):
         return self.as_array().sum(*args, **kwargs)
+    def norm(self):
+        '''return the norm of the DataContainer'''
+        y = self.as_array()
+        return numpy.dot(y, y.conjugate())
```

Comment:
Note that this gives you the SQUARED norm.

Reply:
that's right!